webui/react/src/pages/TrialDetails/TrialRangeHyperparameters.module.scss

```diff
@@ -0,0 +1,98 @@
+$color-blue: rgb(0, 155, 222);
```

Comment:
(this is something i often forget too) we have css vars defining colors and some other variables for measures (margin, border-radius, ...). Would be best to use them here too.

Reply:
eg: this color is `var(--theme-colors-action-normal)`